spec/identity/index.html

```diff
@@ -476,18 +476,18 @@ <h2 id="h2_terminology"><span class="secno">2. </span>Terminology</h2>
 <dd>The Requesting Agent initiates a request to a Service listening on a specific port using a given protocol on a given Server.</dd>
 
 <dt><dfn title="Server" id="dfn-server">Server</dfn></dt>
-<dd>A Server is a machine contactable at a domain name or IP address that hosts a number of globally accessible Services.</dd>
+<dd>A Server is a machine, contactable at a domain name or IP address, that hosts globally accessible Services.</dd>
```

Comment:
```suggestion
<dd>A Server is a machine, contactable at a domain name or IP address, that hosts Services which are accessible over the network. Accessibility may be global, local, or anywhere in between, depending on firewall, service, and related policy configurations.</dd>
```

Reply:
Mhh I wonder how much we want to get into the details here... We should look at why these definitions are there at all. I did not notice them being used that much. WebIDs really work if they are global identifiers. So that is a SHOULD as there is no way to guarantee they are globally available, and it allows for local exceptions.

Reply:
This may be useful to some readers but I think it is cumbersome. If we are to update this definition, I'd suggest to keep the current text as is and then reference "server" as defined in RFC 7230 ( https://www.rfc-editor.org/rfc/rfc7230#section-2.1 ) or RFC 8110 ( https://www.rfc-editor.org/rfc/rfc9110#section-3.3-2 ) - I believe that is what's intended. That can be part of a separate PR.

Reply:
This definition is not of a "WebID Server"; it's of a "Server".

I'm not attached to the specifics of this definition; the part I have issue with is "globally accessible", which I think is not so often the case.

Even if this *is* defining "WebID Server", I don't think "globally accessible" is mandatory; it only needs to be accessible by those the WebID referent wants to access their WebID Document.

I think I would also be fine with removing these definitions of Service and Server, and linking out to relevant RFC, Wikipedia, or other pre-existing definitions in the world.

Reply:
Maybe this is more acceptable to folks --
```suggestion
<dd>A Server is a machine, contactable at a domain name or IP address, that hosts Services which are accessible over the network.</dd>
```